packages/paste-website/src/pages/getting-started/inclusive-design/index.mdx

```diff
@@ -0,0 +1,285 @@
+---
+title: Inclusive Design Guide
+description: Things to consider while building inclusive, accessible experiences for Twilio users.
+slug: /getting-started/inclusive-design/
+---
+
+import {graphql} from 'gatsby';
+import {
+  UnorderedList,
+  Card,
+  ListItem,
+  Stack,
+  Alert,
+  Anchor,
+  Disclosure,
+  DisclosureHeading,
+  DisclosureContent,
+} from '@twilio-paste/core';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {Grid} from '../../../components/grid';
+import {Link} from 'gatsby';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/getting-started/inclusive-design/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## What is this guide?
+
+Designing inclusive and accessible experiences may seem daunting. However, proper application of these practices can make your UX faster, more familiar, and available to the widest possible audience. It's also, quite simply, the right thing to do.
+
+This guide includes a lightly modified version of the excellent [Inclusive Web Design Checklist](https://github.com/Heydon/inclusive-design-checklist) by **Heydon Pickering**. It also has observations and notes from the Design Systems team.
+
+**Keep us accountable:** Track our progress on our [GitHub Discussions thread](https://github.com/twilio-labs/paste/discussions/561) as we apply these principles across this website.
+
+## How to use this guide
+
+We recommend using this guide in the following ways:
+
+- Read through the guide once. We recommend starting with sections that relate to your role. However, reading the entire list will give you a richer understanding of inclusive design and accessibility.
+- Share this guide with your design, product, and engineering peers. Engage with them in a conversation about how we could cave some space in our day-to-day work to make our Twilio experiences a little more inclusive.
```

Comment:
Small typo. "Cave" should be "carve."

Reply:
Thanks! For small typos, feel free to suggest a commit! Select the line and hit the +- icon to make a suggestion!